app/controllers/users_controller.rb

```diff
@@ -36,6 +36,18 @@ def update
     end
   end
 
+  def edit_password
```

Comment:
Technically the #edit_password and #update_password methods are not ReSTful routes, but these wound up working.

Reply:
if it works, it works! maybe we can ask Cory about this at the next project check-in